app/css/LoginStyle.css

```diff
@@ -43,16 +39,12 @@ body {
 }
 
 .input-box input {
-
-    border: none;

```

Comment:
Add a :focus style with a visible outline or other visual indicator for better accessibility accessibility